docs/content/concepts/terminology.md

```diff
@@ -1,100 +1,142 @@
+this terminology before proceeding. Note that some terms might get changed in the future, and that will be covered by
+respective design documents.
+
 ## Logical Cluster
 
-A logical cluster is a way to subdivide a single kube-apiserver + etcd storage into multiple clusters (different APIs,
-separate semantics for access, policy, and control) without requiring multiple instances.  A logical cluster is a
-mechanism for achieving separation, but may be modelled differently in different use cases.  A logical cluster is
-similar to a virtual cluster as defined by sig-multicluster, but is able to amortize the cost of a new cluster to be
-zero or near-zero memory and storage so that we can create tens of millions of empty clusters cheaply.
+A logical cluster is a way to subdivide a single `kube-apiserver` instance and `etcd` datastore into multiple
+clusters (different APIs, separate semantics for access, policy, and control) without requiring multiple instances.
+
+A logical cluster is a mechanism for achieving separation, but may be modelled differently in different use cases.
+A logical cluster is able to amortize the cost of a new cluster to be near-zero memory and storage, so that we can
+create a huge number of empty clusters cheaply.
 
-A logical cluster is a storage level concept that adds an additional attribute to an object’s identifier on a
-kube-apiserver.  Regular servers identify objects by (group, version, resource, optional namespace, name).  A logical
-cluster enriches an identifier: (group, version, resource, **logical cluster name**, optional namespace, name).
+A logical cluster is a storage level concept that adds an additional attribute to an object’s identifier in etcd
+and kube-apiserver. Regular API servers identify objects by (group, version, resource, optional namespace, name).
+A logical cluster enriches an identifier with its name:
+(group, version, resource, **logical cluster name**, optional namespace, name).
 
 ## Workspace
 
-A workspace models a set of user-facing APIs for CRUD.  Each workspace is backed by a logical cluster, but not all
-logical clusters may be exposed as workspaces.  Creating a Workspace object results in a logical cluster being available
-via a URL for the client to connect and create resources supported by the APIs in that workspace.  There could be
-multiple different models that result in logical clusters being created, with different policies or lifecycles, but
-Workspace is intended to be the most generic representation of the concept with the broadest possible utility to anyone
-building control planes.
+A workspace provides Kubernetes-cluster-like HTTPS/CRUD endpoints, i.e., endpoints that typical Kubernetes client
+tooling (e.g., client-go, controller-runtime, and others) can interact with as they would with classic Kubernetes
+clusters. Workspace provides multi-tenancy in a kcp setup.
```

Comment:
This is not correct now, it was before, just maybe misplaced. I would move what you wrote here nearly 1:1 to logical clusters:
```
For each logical cluster, kcp provides  Kubernetes-cluster-like HTTPS/CRUD endpoints, i.e., endpoints that typical Kubernetes client
tooling (e.g., client-go, controller-runtime, and others) can interact with as they would with classic Kubernetes
clusters.
```

Reply:
I kind of copied that from this page: https://docs.kcp.io/kcp/v0.26/concepts/workspaces/

Is that page also incorrect or is my interpretation of it wrong? My understanding is that logical clusters are inaccessible on their own, i.e. that they're only a storage-level concept, but that you need something on top of them (i.e. Workspaces) to get access to them?

Based on this, does it mean that you can create logical clusters that are exposed without a Workspace?

Reply:
Also relevant: https://github.com/kcp-dev/kcp/pull/3238#discussion_r1898595485

Reply:
you can access logical clusters directly, if you know their name. Long ago, we need RBAC to give access to the logical cluster from the parent. At that time we needed the workspace. This is not the case anymore.